sale_manual_delivery/models/sale_order.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# Copyright 2021 Iván Todorovich, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    manual_delivery = fields.Boolean(
+        string="Manual Delivery",
+        default=False,
+        help="If Manual, the deliveries are not created at SO confirmation. "
```

Comment:
```suggestion
        help="If True, the deliveries are not created at SO confirmation. "
```
nitpicking: since boolean can have only True or False, Manual sounds more like a option for Selection field to me.